app/install/centos.md

```diff
@@ -7,60 +7,43 @@ breadcrumbs:
   Installation: /install
 ---
 
-## Packages
-
-Start by downloading the corresponding package for your configuration:
-
-- [CentOS 6]({{ site.links.download }}/kong-rpm/download_file?file_path=centos/6/kong-{{site.data.kong_latest.version}}.el6.amd64.rpm)
-- [CentOS 7]({{ site.links.download }}/kong-rpm/download_file?file_path=centos/7/kong-{{site.data.kong_latest.version}}.el7.amd64.rpm)
-- [CentOS 8]({{ site.links.download }}/kong-rpm/download_file?file_path=centos/8/kong-{{site.data.kong_latest.version}}.el8.amd64.rpm)
-
-**Enterprise trial users** should download their package from their welcome email and save their license to `/etc/kong/license.json` after step 1.
-
-### YUM Repositories
-
-You can also install Kong via YUM; follow the instructions on the "Set Me Up"
-section on the page below.
-
-- [RPM Repository](https://bintray.com/kong/kong-rpm)
-
-**NOTE**: ensure that the `baseurl` field of the generated `.repo` file contains
-your CentOS version; for instance:
-
-```
-baseurl=https://kong.bintray.com/kong-rpm/centos/6
-```
-or
-
-```
-baseurl=https://kong.bintray.com/kong-rpm/centos/7
-```
-
-----
-
 ## Installation
 
-1. **Install Kong**
-
-    If you are downloading the [package](#packages), execute:
-
+1. **Installation**
+
+    Kong can be installed by downloading an installation package or from our yum repository
+    
+    **Packages**
```

Comment:
Maybe we can put these sections into tabs? Tabs are nice for either/or situations like this one.

Here's what it looks like after a quick test:
![Screen Shot 2021-04-13 at 5 08 36 PM](https://user-images.githubusercontent.com/54370747/114636020-e7924c80-9c7a-11eb-8a40-32f07f00aa7a.png)

Navtab code:
```
{% navtabs %}
{% navtab Download packages %}

content

{% endnavtab %}
{% navtab YUM repo %}

content

{% endnavtab %}
{% endnavtabs %}
```